src/test/java/com/google/sps/servlets/DataServletTest.java

```diff
@@ -101,14 +113,15 @@ public void addTermRating_newRating() throws IOException {
   public void addTermRating_overwritingExistingTermRating() throws IOException {
     // SETUP.
     // File with body request in webapp folder.
+    HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
     Reader originalRatingReader = new FileReader("src/main/webapp/WEB-INF/testNewRating.txt");
     BufferedReader originalRatingBufferedReader = new BufferedReader(originalRatingReader);
     when(request.getReader()).thenReturn(originalRatingBufferedReader);
 
-    Float sentimentScore = (float) -0.699999988079071;
+    double sentimentScore = (double) -0.699999988079071;
     AnalyzeSentimentResponse response =
         AnalyzeSentimentResponse.newBuilder()
-            .setDocumentSentiment(Sentiment.newBuilder().setScore(sentimentScore))
+            .setDocumentSentiment(Sentiment.newBuilder().setScore((float) -0.699999988079071))
```

Comment:
why create the sentimentScore variable if you are not using it here?

Reply:
Yeah on this it was too describe the variable better but I think I will just use the value directly, will add a comment just in case. Thanks!!